ecchronos-binary/src/bin/ecctool.py

```diff
@@ -1,5 +1,4 @@
-#!/bin/sh
-""""exec python -B -- "$0" ${1+"$@"} # """
+#!/usr/bin/env python3
```

Comment:
Probably need to target an older version and merge forward

Reply:
I don't think we need to since it's now we only support python3. Previously we supported both python 2 and python 3 and the soft-link python always pointed to python 2.

Reply:
Sounds reasonable